crates/core/src/db/datastore/system_tables.rs

```diff
@@ -109,10 +110,15 @@ macro_rules! st_fields_enum {
                 }
             }
         }
+
+        impl From<$ty_name> for NonEmpty<ColId> {
+            fn from(value: $ty_name) -> Self {
+                NonEmpty::new(value.col_id())
+            }
+        }
     }
 }
 
-// WARNING: In order to keep a stable schema, don't change the discriminant of the fields
```

Comment:
Why did we lose these comments?

Reply:
I see them inside the macro `st_fields_enum`